README.md

```diff
@@ -1,49 +1,97 @@
+on [Steam](https://store.steampowered.com/bundle/39394). Once the game is ready, download the latest version of the
+project from [GitHub Releases](https://github.com/TheSuperHackers/GeneralsGameCode/releases), extract the necessary 
+files, and follow the instructions in the [Wiki](https://github.com/TheSuperHackers/GeneralsGameCode/wiki).
 
-NOTE: As modern versions of MSVC enforce newer revisions of the C++ standard, you will need to make extensive changes to the codebase before it successfully compiles, even more so if you plan on compiling for the Win64 platform.
+### Disclaimer
 
-When the workspace has finished building, the compiled binaries will be copied to the folder called `/Run/` found in the root of each games directory. 
+The current builds may have bugs or unexpected behavior. We will aim to release stable versions that are as free from
+bugs as possible, but even those may have some unexpected behaviors.
 
+### Antivirus Warning
 
-## Known Issues
+Some antivirus software may mistakenly flag this project as a false positive (e.g., Windows Defender). We assure you
+that the project is fully safe to use. However, if you have concerns, you can
+always [build it yourself](#building-the-game-yourself).
 
-Windows has a policy where executables that contain words “version”, “update” or “install” in their filename will require UAC Elevation to run. This will affect “versionUpdate” and “buildVersionUpdate” projects from running as post-build events. Renaming the output binary name for these projects to not include these words should resolve the issue for you.
+## Joining the Community
 
+You can chat and discuss the development of the project on our
+[Discord channel](https://www.community-outpost.com/discord) or on 
+our [GameReplays forum](https://www.gamereplays.org/community/index.php?showforum=132). Join us to get the latest
+updates, report bugs, and contribute to the project!
 
-## STLport
-STLport will require changes to successfully compile this source code. The file [stlport.diff](stlport.diff) has been provided for you so you can review and apply these changes. Please make sure you are using STLport 4.5.3 before attempting to apply the patch.
+## Building the Game Yourself
 
+We provide support for building the project using Visual Studio 6 (VS6) and Visual Studio 2022. For detailed build
+instructions, check the [Wiki](https://github.com/TheSuperHackers/GeneralsGameCode/wiki/build_guides), which also
+includes guides for building with Docker, CLion, and links to forks supporting additional versions.
 
-## License
+## Contributing
 
-This repository and its contents are licensed under the GPL v3 license, with additional terms applied. Please see [LICENSE.md](LICENSE.md) for details.
+We welcome contributions to the project! If you’re interested in contributing, you should have some knowledge of C++ and
+Visual Studio. We also allow this to be built without Visual Studio. Join the developer chat on Discord for more
```

Comment:
I do not think it should say "We also allow this to be built without Visual Studio.", because why would we disallow it in the first place?